WordPress/Classes/ViewRelated/Reader/ReaderReblogAction+Formatter.swift

```diff
@@ -0,0 +1,69 @@
+/// Contains methods to format post reblog content for either Gutenberg (block) editor or Aztec (classic) editor
+struct ReblogFormatter {
+
+    static func gutenbergQuote(text: String, citation: String? = nil) -> String {
+        let quote = quoteWithCitation(text: text, citation: citation)
+        return embedInWpQuote(html: quote)
+    }
+
+    static func gutenbergImage(image: String) -> String {
+        let imageInHtml = htmlImage(image: image)
+        return embedInWpParagraph(html: imageInHtml)
+    }
+
+
+    static func aztecQuote(text: String, citation: String? = nil) -> String {
+        let quote = quoteWithCitation(text: text, citation: citation)
+        return embedInQuote(html: quote)
+    }
+
+    static func aztecImage(image: String) -> String {
+        let imageInHtml = htmlImage(image: image)
+        return embedInParagraph(html: imageInHtml)
+    }
+}
+
+
+// MARK: - Gutenberg formatter helpers
+extension ReblogFormatter {
```

Comment:
Just a suggestion - if this were a `private` extension the methods wouldn't need to be explicitly `private`.

Reply:
Done!